<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editDisplayMetaItem</class>
 <widget class="QDialog" name="editDisplayMetaItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>editDisplayMetaItem</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TAG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QIndexComboBox" name="m_cboEditorTag"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbTagApply">
       <property name="text">
        <string>ApplyTag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QIndexComboBox" name="m_cboImageListIndex"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Pivot Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cboCenter"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="m_ctlX"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="m_ctlY"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Position easing type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cboEasing"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rotation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="m_ctlAngle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Angle easing type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cboEasing_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="m_ctlHeight"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="m_ctlWidth"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Coloring</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QColorPickerPushButton" name="m_ctlImageColor">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Color easing type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cboEasing_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Horizontal Align</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cboHA"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Vertical Align</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cboVA"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Z-order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_ctlZOrder"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QCheckBox" name="m_cbLimitBound">
       <property name="text">
        <string>Limit to boundary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_cbVisible">
       <property name="text">
        <string>Visible</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QColorPickerPushButton</class>
   <extends>QPushButton</extends>
   <header>qcolorpickerpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QIndexComboBox</class>
   <extends>QPushButton</extends>
   <header location="global">qindexcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
